telegram/_message.py

```diff
@@ -3096,7 +3096,7 @@ async def reply_game(
             do_quote, quote, reply_to_message_id, reply_parameters
         )
         return await self.get_bot().send_game(
-            chat_id=chat_id,
+            chat_id=int(chat_id),
```

Comment:
I would vote for a `# type: ignore[arg-type]` instead, here. My reasoning:

* `chat_id` is annotated here as `str | int` as return value of the internal `_parse_quote_arguments`, which is supposed to be used by all shortcut methods, even those that accept strings for `chat_id`. So forcing `_parse_quote_arguments` to return an integer doesn't make sense
* However, we generally do not enforce type checks before passing values to Telegram. If a user would pass a string here, Telegram will through a descriptive error message if it can't parse it. That should be enough